### PR TITLE
Documentation for the Versioning package

### PR DESCRIPTION
**Pull request by JohnTortugo**
Closes: #480 

Some documentation around the signing package in the Arcade SDK.

### CODE REVIEW

File: Documentation/CorePackages/VersioningPackage.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This is the list of output properties from the Versioning package.
+
+| Name            | Description                                                  |
+| --------------- | ------------------------------------------------------------ |
+| PackageVersion  | Append short commit SHA to PackageVersion. SemanticVersioningV1 controls the separator between the two fields. |
+| AssemblyVersion | Set to "42.42.42.42" if empty and VersionSuffixDateStamp is also empty, otherwise leave as is. |
+| FileVersion     | FileVersion string. If "VersionSuffixDateStamp" is empty FileVersion will be set to "42.42.42.42424". |
+| VersionPrefix   | The leading part of the version string as specified in the Versioning schema. |
+| VersionSuffix   | The suffix part of the version string, including the pre-release portion. |
+
```
Comment (jcagme) [inline Documentation/CorePackages/VersioningPackage.md]: Remove all blank lines at the bottom

**Comment by tmat** [comment]
The versioning is not a package.

Also, I think this doc duplicates some information documented [here](https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#project-properties-defined-by-the-sdk) and also documents properties that are not defined by Arcade, but by .NET SDK.

**Comment by tmat** [comment]
Maybe we should just reword - we can use this page to document all version related settings. I'd be then useful to point out which ones are in .NET SDK and which ones are Arcade specific.

**Comment by JohnTortugo** [comment]
I think that is a good suggestion. I'm working around it.

**Comment by JohnTortugo** [comment]
@tmat I updated following your suggestion. I'm not sure if I got clear the idea of distinguishing if the parameter is from .NET or Arcade.. PTAL.

**Pull request merged**